source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -862,15 +863,17 @@ int ClientContextImpl::newSessionKey(SSL_SESSION* session) {
   return 1; // Tell BoringSSL that we took ownership of the session.
 }
 
-uint16_t ClientContextImpl::parseSigningAlgorithmsForTest(const std::string& sigalgs) {
+absl::optional<uint16_t>
```

Comment:
Please do not mix code changes into this PR

Reply:
Reverted this change.